daisyui_form_builder.rb

```diff
@@ -0,0 +1,313 @@
+class AppFormBuilder < ActionView::Helpers::FormBuilder
+  delegate :tag, :safe_join, to: :@template
+
+  def input(method, options = {})
+    @form_options = options
+    object_type = object_type_for_method(method)
+
+    input_type = if options[:as]
+      options[:as]
+    elsif options[:collection]
+      :select
+    else
+      object_type
+    end
+
+    override_input_type = case input_type
+    when :integer then :string
+    when :email then :string
+    when :telephone then :string
+    when :url then :string
+    when :datetime_local then :string
+    when :password then :string
+    when :search then :string
+    when :week then :string
+    when :month then :string
+    when :date then :datetime
+    when :time then :datetime
+    end
+
+    send("#{override_input_type || input_type}_input", method, options)
+  end
+
+  def self.with_blank_error_proc(&block)
+    old_error_proc = ActionView::Base.field_error_proc
+    begin
+      ActionView::Base.field_error_proc = proc do |tag, instance|
+        tag
+      end
+      block.call
+    ensure
+      ActionView::Base.field_error_proc = old_error_proc
+    end
+  end
+
+  # def label(method, label = nil, options = {}, &)
+  #   super(method, label, merge_input_options({class: "form-label"}, options), &)
+  # end
+
+  private
+
+  def form_group(method, options = {}, &block)
+    label(method, class: "form-control mb-3") do
+      safe_join [
+        label_text(method, options[:label], class: "font-bold"),
+        block.call,
+        hint_text(options[:hint]),
+        error_text(method)
+      ].compact
+    end
+  end
+
+  def collection_group(method, options = {}, &block)
+    safe_join [
+      label_text(method, options[:label], class: "font-bold"),
+      block.call,
+      hint_text(options[:hint]),
+      error_text(method)
+    ].compact
+  end
+
+  def label_text(method, text = nil, options = {})
+    return if text == false
+    text ||= method.to_s.humanize
+    tag.div(class: "label #{options[:class]}") do
```

Comment:
If we want all labels to be bold, maybe we do that here instead of passing the class in? `label font-bold #{options[:class]}`